Smartpad-iOS/Smartpad-iOS/Logic/HapticManager.swift

```diff
@@ -26,38 +26,73 @@ class HapticManager {
 }
 
 extension HapticManager {
-    private func slicePattern() throws -> CHHapticPattern {
-    let slice = CHHapticEvent(
-      eventType: .hapticContinuous,
-      parameters: [
-        CHHapticEventParameter(parameterID: .hapticIntensity, value: 0.35),
-        CHHapticEventParameter(parameterID: .hapticSharpness, value: 0.25)
-      ],
-      relativeTime: 0,
-      duration: 0.25)
+    private func touchDownPattern() throws -> CHHapticPattern {
+        let slice = CHHapticEvent(
```

Comment:
This doesn't feel quite right, I think maybe the commented out code is correct?

Reply:
I don't quite understand what you mean by "feel". Do you mean the physical feedback?

Reply:
When I checked out this commit, I feel a vibration when pressing down and a "click" when releasing. I think the original intent was to have a "click" when pressing down, then another when releasing.

So basically right now we have:
press down - vibration/rolling feeling
release - click up feeling

I expected:
press down - click down feeling
release - click up feeling

If you uncomment the snip and comment out the slice for touch down, it feels more close to how I would expect

Reply:
I see, so you mean to have the two feedbacks be similar? I thought it was more appropriate to differentiate them with noticably different feeling. Either is alright though, it's just change a few values.

Reply:
well sort of like the flashlight button, there is one feeling for pressing down and another for releasing, they feel different but one feels like pressing a button down and the other feels like releasing the button.